toonz/sources/include/tsound_t.h

```diff
@@ -85,9 +85,9 @@ class DVAPI TSoundTrackT final : public TSoundTrack {
   //----------------------------------------------------------------------------
 
   /*!
-Returns a soundtrack whom is a clone of the object for the spicified channel
-A clone means that it's an object who lives indipendently from the other
-from which it's created.It hasn't reference to the object.
+Returns a soundtrack whom is a clone of the object for the spicified channel.
```

Comment:
The automated typo fix here failed at the word 'spicified' but seems to have fixed the remainder of the paragraph.

Reply:
Will update the codespell dict to add `spicified`! Thanks. Fixed in 69208c7